scripts/libwebp/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+export CC=${FAM_CC}
```

Comment:
Please pass the compiler to the configure script directly. The build.sh files are executed with the `source` command, which may modify the outer environment and the CC variable may affect all subsequent build scripts (even for other ABIs).

Reply:
> Please pass the compiler to the configure script directly. The build.sh files are executed with the `source` command, which may modify the outer environment and the CC variable may affect all subsequent build scripts (even for other ABIs).

I see, i will change that

Reply:
Done 388294b

Reply:
I actually ment passing the CC directly to the configure script like [this](https://github.com/Javernaut/ffmpeg-android-maker/blob/master/scripts/libfreetype/build.sh#L21).

Reply:
> I actually meant passing the CC directly to the configure script like [this](https://github.com/Javernaut/ffmpeg-android-maker/blob/master/scripts/libfreetype/build.sh#L21).

my bad, i changed it now